example/src/test/scala/monocle/std/ValidationExample.scala

```diff
@@ -0,0 +1,42 @@
+package monocle.std
+
+import org.specs2.scalaz.{ScalazMatchers, Spec}
+
+import monocle.std.{validation => mValidation}
+
+import scalaz.syntax.validation._
+import scalaz.syntax.either._
+
+class ValidationExample extends Spec {
+  "success defines a Prism that can get, set or modify the underlying value of a Success instance" in {
+    mValidation.success.getOrModify(123.success) ==== 123.right
+    mValidation.success.getOrModify("abc".failure) ==== "abc".failure.left
+
+    mValidation.success.getOption(123.success) ==== Some(123)
```

Comment:
you may want to limit the example to one or two prism methods e.g. `getOption` and `set` or `modify`. The laws will do the extensive tests